keyboards/idobao/id96/id96_via_sideload_kle.json

```diff
@@ -0,0 +1,623 @@
+{
```

Comment:
Needs to be removed, we do not accept VIA config files to this repo.

Reply:
deleted with commit 8e4ec92